### PR TITLE
Add liveview based autocomplete for partner search

### PR DESCRIPTION
**Pull request by ashkan18**
#  Live View Based Autocomplete
For finding partners,

# Selfie
image

# Next Improvement
Throttle requests or have limits for more than few characters

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -7,29 +7,42 @@
...
+  api_url: System.get_env("GRAVITY_API_URL") || "https://stagingapi.artsy.net",
   api_token: System.get_env("GRAVITY_API_TOKEN")
 
 config :artsy_auth_ex,
-  token_aud: System.get_env("ARTSY_TOKEN_AUD"), # aud of your JWT token, Gravity's ClientApplication.id
-  client_id: System.get_env("ARTSY_CLIENT_ID"), # Gravity's ClientApplication.app_id
-  client_secret: System.get_env("ARTSY_CLIENT_SECRET"), # Gravity's ClientApplication.app_secret
-  redirect_uri: Map.get(System.get_env(), "HOST_URL", "http://localhost:4000") <> "/auth/callback",
-  site: System.get_env("ARTSY_URL"), # Gravity's api url ex. https://stagingapi.artsy.net
+  # aud of your JWT token, Gravity's ClientApplication.id
```
Comment (ashkan18) [inline config/config.exs]: ignore these, all `mix format` magic

File: lib/inflow/import.ex [hunk 2]
```diff
@@ -30,7 +30,8 @@ defmodule Inflow.Import do
   """
   def start_upload(file_path, file_name, partner_id, is_auction \\ false) do
     # upload to S3
-    with {:ok, manifest} <- create_manifest(%{partner_id: partner_id, is_auction: is_auction, file_name: file_name}),
+    with {:ok, manifest} <-
```
Comment (ashkan18) [inline lib/inflow/import.ex]: we may want to increase line length for `format` 🤷‍♂

File: lib/inflow_web/router.ex [hunk 3]
```diff
@@ -20,7 +21,8 @@ defmodule InflowWeb.Router do
   scope "/", InflowWeb do
     pipe_through [:browser, :authenticated]
 
-    get "/", PageController, :index
+    live "/", DashboardView, session: [:access_token]
```
Comment (ashkan18) [inline lib/inflow_web/router.ex]: creates a new liveview page mounted on / and passes current session's access_token to the live view.

File: lib/inflow_web/controllers/manifests_controller.ex [hunk 4]
```diff
@@ -1,9 +1,13 @@
 defmodule InflowWeb.ManifestsController do
   use InflowWeb, :controller
   alias Inflow.{Import, Import.Manifest}
+  @gravity_api Application.get_env(:inflow, :gravity_api)
 
-  def index(conn, _params) do
-    render(conn, "index.html", manifests: Import.list_manifests())
+  def index(conn, %{"partner_id" => partner_id}) do
+    partner = @gravity_api.get!("/api/v1/partner/#{partner_id}").body
```
Comment (bhoggard) [inline lib/inflow_web/controllers/manifests_controller.ex]: I can't tell. Does this `get` happen every time this page is accessed, even if it's in the session?

Comment (ashkan18) [reply lib/inflow_web/controllers/manifests_controller.ex]: yes, we don't add `partner` to the session though, we only add `partner_id`.

**Pull request merged**